packages/circle-demo-webapp/app/lib/types.ts

```diff
@@ -0,0 +1,757 @@
+/**
+ * Request types
+ */
+import {
+  FeeLevel,
+  TransactionState,
+  WalletState,
+} from '@circle-fin/developer-controlled-wallets';
+
+import { Blockchain, TestnetBlockchain, TransactionType } from '~/lib/constants';
+
+export type TypeBlockchain = (typeof Blockchain)[keyof typeof Blockchain];
+export type TypeTestnetBlockchain =
+  (typeof TestnetBlockchain)[keyof typeof TestnetBlockchain];
+export type TypeTransactionType = (typeof TransactionType)[keyof typeof TransactionType];
+
+export enum TransferState {
+  CANCELLED = 'CANCELLED',
+  CONFIRMED = 'CONFIRMED',
+  COMPLETE = 'COMPLETE',
+  DENIED = 'DENIED',
+  FAILED = 'FAILED',
+  INITIATED = 'INITIATED',
+  PENDING_RISK_SCREENING = 'PENDING_RISK_SCREENING',
+  QUEUED = 'QUEUED',
+  SENT = 'SENT',
+}
+
+/**
+ * A developer controlled wallet set
+ * https://developers.circle.com/api-reference/w3s/developer-controlled-wallets/create-wallet-set
+ * https://developers.circle.com/api-reference/w3s/developer-controlled-wallets/update-wallet-set
+ * https://developers.circle.com/api-reference/w3s/developer-controlled-wallets/get-wallet-sets
+ * https://developers.circle.com/api-reference/w3s/developer-controlled-wallets/get-wallet-set
+ */
+export interface WalletSet {
```

Comment:
wait, doesn't circle sdk export type definitions? I will check what they have then.

Reply:
unfortunately, they do not export types.